depthai_ros_driver/src/pipelines/stereo_pipeline.cpp

```diff
@@ -19,7 +19,7 @@ class StereoPipeline : public rr::Pipeline {
     void onConfigure(const std::string& config_json) {
         ROS_WARN("For running this stereo example pipeline, "
                  "Please specify \"stream_list\" as "
-                 "\"[left, right, disparity, depth, rectified_left, rectified_right]\". "
+                 "\"[left, right, depth, rectified_left, rectified_right]\". "
```

Comment:
Or something like: please ensure not to request for `depth` stream alongside `disparity` or `disparity_color` or both

Reply:
The issue is that the system hang up if a stream setup in the pipeline is not read out from the driver. 
I meant `StereoPipeline` to be an example of a pipeline just copying `stereo_example.cpp` by limiting the usage. So, I think we can leave more flexible usage to the plugin developers as long as we have more flexible `MultiCameraPipeline`.